src/locales/en-US/commands.json

```diff
@@ -303,7 +303,7 @@
     "commandDescription": "Searches for an anime on MyAnimeList",
     "commandUsage": "<anime name>",
     "noAnime": "You need to give me an anime to search for",
-    "score": "Score",
+    "score": "Score rating",

```

Comment:
I'd say „Score” should stay as it is, since it's „Score” on MAL too. If we want to be really specific, it's a „Weighted score”.

Reply:
Hmm, yeah. Took a look, and on MAL they both refer to it as either a "Score" or just outright "Rating"